model/events/parse.go

```diff
@@ -31,7 +31,7 @@ func ParseEvent(eventType flow.EventType) (*ParsedEvent, error) {
 	parts := strings.Split(string(eventType), ".")
 
 	switch parts[0] {
-	case "flow":
+	case "flow", "evm":
```

Comment:
better to reference the https://github.com/onflow/flow-go/blob/master/fvm/evm/types/events.go#L20

Reply:
@sideninja That's a good point, but it is not an exported constant. Should I export it?

Reply:
yeah go ahead